bropkg/composer.json

```diff
@@ -5,16 +5,17 @@
     "type": "project",
     "license": "MIT",
     "require": {
-        "php": ">=5.6",
-        "cakephp/cakephp": "3.5.*",
-        "mobiledetect/mobiledetectlib": "2.*",
-        "cakephp/migrations": "~1.0",
+        "php": ">=7.1",
```

Comment:
Isn't PHP 7.1 still over 5 years unmaintained? Better than 5.6 though

Reply:
It is. Moving up to 8.x required a ton of changes to make the Cake library stuff work, and I didn't want to undertake any of that here.